qlib/pagetable.rs

```diff
@@ -751,16 +751,10 @@ impl PageTables {
         }
     }
 
-    pub fn Unmap(&self, start: u64, _end: u64, pagePool: &Allocator) -> Result<()> {
+    pub fn Unmap(&self, start: u64, end: u64, pagePool: &Allocator) -> Result<()> {
         Addr(start).PageAligned()?;
-        Addr(_end).PageAligned()?;
+        Addr(end).PageAligned()?;
 
```

Comment:
As an alternative, we could add a software defined bit in the PTE, so that it can be skipped.